components/sup/src/lib.rs

```diff
@@ -300,10 +300,9 @@ lazy_static!{
 
 features! {
     pub mod feat {
-        const List = 0b00000001,
-        const Multi = 0b00000010
+        const List = 0b00000001
     }
 }
 
-const PRODUCT: &'static str = "hab-sup";
-const VERSION: &'static str = env!("CARGO_PKG_VERSION");
+pub const PRODUCT: &'static str = "hab-sup";
+pub const VERSION: &'static str = env!("CARGO_PKG_VERSION");
```

Comment:
Was changing the implementation here on purpose (this won't put the `build.rs`-driven version string into the binary)?